a8cteam51/onbeing
# Prevent posts that are excluded in search from appearing in the Podcast feeds

This makes it so that when someone checks "Exclude from Search" on the edit episode page, the episode no longer appears in the podcast's XML feed.

I tested these on obtester.com by excluding some posts from search and then visiting the feeds. I think it would be worth someone testing these again on obtester (after this is merged in).

To test:
1. Open any Episode on obtester.com
2. Click "Exclude from Search" in the right column
3. Open up the podcast XML feeds below. The post should NOT appear.

Feeds to test:
https://obtester.com/feed/podcast/podcast_mobile
https://obtester.com/feed/podcast/podcast
https://obtester.com/feed/podcast/becoming-wise
https://obtester.com/feed/podcast/episodes_only
https://obtester.com/feed/podcast/extras_only
